patches/axios+1.6.7.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/node_modules/axios/dist/node/axios.cjs b/node_modules/axios/dist/node/axios.cjs
+index 9099d87..7104f6e 100644
+--- a/node_modules/axios/dist/node/axios.cjs
++++ b/node_modules/axios/dist/node/axios.cjs
+@@ -370,9 +370,9 @@ function merge(/* obj1, obj2, obj3, ... */) {
+ const extend = (a, b, thisArg, {allOwnKeys}= {}) => {
+   forEach(b, (val, key) => {
+     if (thisArg && isFunction(val)) {
+-      a[key] = bind(val, thisArg);
++      Object.defineProperty(a, key, {value: bind(val, thisArg)});
+     } else {
+-      a[key] = val;
++      Object.defineProperty(a, key, {value: val});
+     }
+   }, {allOwnKeys});
+   return a;
+@@ -403,7 +403,9 @@ const stripBOM = (content) => {
+  */
+ const inherits = (constructor, superConstructor, props, descriptors) => {
+   constructor.prototype = Object.create(superConstructor.prototype, descriptors);
+-  constructor.prototype.constructor = constructor;
++  Object.defineProperty(constructor, 'constructor', {
++    value: constructor
++  });
+   Object.defineProperty(constructor, 'super', {
+     value: superConstructor.prototype
+   });
+@@ -565,12 +567,14 @@ const isRegExp = kindOfTest('RegExp');
+ 
+ const reduceDescriptors = (obj, reducer) => {
+   const descriptors = Object.getOwnPropertyDescriptors(obj);
+-  const reducedDescriptors = {};
++  let reducedDescriptors = {};
+ 
+   forEach(descriptors, (descriptor, name) => {
+     let ret;
+     if ((ret = reducer(descriptor, name, obj)) !== false) {
+-      reducedDescriptors[name] = ret || descriptor;
++      reducedDescriptors = {...reducedDescriptors, 
++        [name]: ret || descriptor
++      };
```

Comment:
FFR, we should probably use `Object.defineProperty(reducedDescriptors, name, { value: ret || descriptor })` since otherwise this loop is quadratic.

Reply:
That's what I did first but when PRing upstream it caused a test failure.